packages/react/src/components/Geo/MapView/index.tsx

```diff
@@ -96,6 +96,9 @@ const MapView = forwardRef<MapRef, MapViewProps>(
         ref={ref}
         style={styleProps}
         transformRequest={transformRequest}
+        // workaround until type is fixed on react-map-gl: https://github.com/visgl/react-map-gl/issues/1973
+        fog={props.fog ?? undefined}
```

Comment:
I think the react unit tests might be failing because this need a unit test to bump coverage back to 80%

Reply:
Added unit test for mapview to bump up coverage